conftest.py

```diff
@@ -1,12 +1,17 @@
-# Configuration file for running contract-tests
 import configparser
 import pytest
 import ssl
 
+from dotenv import load_dotenv, find_dotenv
+
+
 # Hack because of how SSL certificates are verified by default in Python
 if hasattr(ssl, '_create_unverified_context'):
     ssl._create_default_https_context = ssl._create_unverified_context
 
+# Load our dotenv file
+load_dotenv(find_dotenv())
```

Comment:
If you're running the tests with pipenv, then `.env` files are already loaded into the environment and you shouldn't need to install/user python-dotenv. See https://docs.pipenv.org/advanced/#automatic-loading-of-env for details.

Reply:
Fixed